### ISSUE TITLE
Connecting with home assistant (hass.io)?

### ISSUE THREAD

**Issue by fodi666** [universam1/iSpindel#179]
Hi,

I would like to connect the iSpindel with my Home assistant server. They offer a RESTful API to integrate customs sensors and it seems that the iSpindel itself can send JSON in the TCP mode. However I can't connect them, if I set the server on the iSpindel to my Home assitant server and port, it gives me "HTTP 400 Bad request" on the serial port. 
What address should I call on my server to get the correct JSON out of the iSpindel? I have tried http://ispindel-ip:port/api/hydrometer/v1/data but could not reach it.

Thanks

**Comment by fodi666** [comment]
Finally made it by adding MQTT forward code to the TCP Server by Tozzi.

**Comment by jip67** [comment]
I just made an integration using Node-Red and Brewfather.  I use brewfather as brewing software and there you can attach Ispindel. 
Brewfather can post the Ipindel data using webhook. 
In Home assistant I added the HACS component for node-red.
In node-red I added the webhook node. I created a few sensors and used them in my lovelace card: 
![image](https://user-images.githubusercontent.com/9585214/82044896-7767f900-96ae-11ea-9776-a7361fb319d6.png)

**Comment by Rotjong** [comment]
Can you tell me how you connected ispindel / brewfather to HACS./ Home Assistant ?

**Comment by FordMustang0288** [comment]
> Can you tell me how you connected ispindel / brewfather to HACS./ Home Assistant ?

I am attempting to figure this out for myself but honestly I wouldn't mind a bit more info on the node side of things too.

**Comment by jip67** [comment]
Sure!

I made 2 integrations. This is about the integration with Brewfather.

First I have a nabucasa subsciption and there I created an webhook.
I installed Node-Red on my home assistant. There I am using a webhook node. This node receives the information from Brewfather.
![image](https://user-images.githubusercontent.com/9585214/87660522-7cedc800-c75f-11ea-8856-a72b8ec3efe9.png)

The set.msg.payload.battery part is a 'change payload'
![image](https://user-images.githubusercontent.com/9585214/87660834-feddf100-c75f-11ea-98b1-ed12a1a21cef.png)
 node:: $round(msg.payload.battery, 1)

And using a HACS Node-Red custom integration, I created 4 sensors in HA:
![image](https://user-images.githubusercontent.com/9585214/87662268-3a79ba80-c762-11ea-99ac-975844747a92.png)

Hope this helps.

**Comment by FordMustang0288** [comment]
@jip67 Thank you very much for the detailed response.
My original goal was to make a simple node dashboard as a way to learn. However, I tried to follow your lead instead.

I ran into trouble almost immediately as I needed to install HACS. I'll be honest that I always get confused about the different versions of Home Assistant but I used [this](https://github.com/whiskerz007/proxmox_hassos_install)as a way of getting Home Assistant set up.

I managed to find the configurations.yaml location but I don't have access to curl and/or wget to download the zip file as the CLI uses /bin/ash (not bash) as a shell. I am working on installing an SSH add on to see if I can then proceed with the HACS portion. As of right now my nodes just tell me they won't work until I install that.

I will likely have a few more questions if I can get past this. Thanks again.
